node/src/chain/fsm.rs

```diff
@@ -248,7 +251,7 @@ impl<N: Network, DB: database::DB, VM: vm::VMExecution> SimpleFSM<N, DB, VM> {
         &mut self,
         quorum: &payload::Quorum,
         msg: &Message,
-    ) -> anyhow::Result<()> {
+    ) -> anyhow::Result<Option<Block>> {
```

Comment:
In the description above, we should add:

`It returns the corresponding candidate block if it gets accepted`